content/en/blog/_posts/2022-12-29-scalable-job-tracking-ga/index.md

```diff
@@ -0,0 +1,148 @@
+---
+layout: blog
+title: "Kubernetes 1.26: Job Tracking, to Support Massively Parallel Batch Workloads, Is Generally Available"
+date: 2022-12-29
+slug: "scalable-job-tracking-ga"
+---
+
+**Authors:** Aldo Culquicondor (Google)
+
+The Kubernetes 1.26 release includes a stable implementation of the [Job](/docs/concepts/workloads/controllers/job/)
+controller that can reliably track a large amount of Jobs with high levels of
+parallelism. [SIG Apps](https://github.com/kubernetes/community/tree/master/sig-apps)
+and [WG Batch](https://github.com/kubernetes/community/tree/master/wg-batch)
+have worked on this foundational improvement since Kubernetes 1.22. After
+multiple iterations and scale verifications, this is now the default
+implementation of the Job controller.
+
+Paired with the Indexed [completion mode](/docs/concepts/workloads/controllers/job/#completion-mode),
+the Job controller can handle massively parallel batch Jobs, supporting up to
+100k concurrent Pods.
+
+The new implementation also made possible the development of [Pod failure policy](/docs/concepts/workloads/controllers/job/#pod-failure-policy),
+which is in beta in the 1.26 release.
+
+## How do I use this feature?
+
+To use Job tracking with finalizers, upgrade to Kubernetes 1.25 or newer and
```

Comment:
Would it make sense to link this here?
```suggestion
To use [Job tracking with finalizers](/docs/concepts/workloads/controllers/job/#job-tracking-with-finalizers), upgrade to Kubernetes 1.25 or newer and
```

Reply:
There is nothing in that doc that is not already explained here and we intend to remove the section in 1.28. So I think it's not worth linking.

Reply:
Fair enough.

Reply:
Alright!